docs/develop/tools/contract-verifications.md

```diff
@@ -7,3 +7,18 @@ Contract verification refers to the process of auditing and verifying the code o
 
 It is typically performed by independent auditors, security experts, or automated tools. The process involves a thorough analysis of the contract code to identify potential issues, such as bugs, security vulnerabilities, or potential attacks. This can involve both manual and automated testing to ensure that the code meets certain standards and best practices.
 
+## [EScan](https://escan.live/)
```

Comment:
I saw that they spell it differently, when you look at the tab title on https://escan.live/

```suggestion
## [Escan](https://escan.live/)
```